frontend/src/features/mapSideBar/hooks/useMapSideBarQueryParams.tsx

```diff
@@ -98,7 +104,12 @@ export const useMapSideBarQueryParams = (map?: L.Map): IMapSideBar => {
         break;
       case MapViewState.PROPERTY_INFORMATION:
         setSidebarComponent(
-          <MotiInventoryContainer onClose={handleClose} pid={propertyId} onZoom={onZoom} />,
+          <MotiInventoryContainer
+            onClose={handleClose}
+            id={!!propertyId ? +propertyId : undefined}
```

Comment:
this looks very odd. Can this logic be done when setting the propertyId variable instead?

Reply:
yes, that works